install.sh

```diff
@@ -3973,6 +4043,76 @@ install_into_existing_repositories() {
     return
 }
 
+############################################################
+# Install the new Git hook templates into
+#   all repos registered for autoupdate.
+#
+# Returns: 0
+############################################################
+install_into_registered_repositories() {
+
+    LIST="$INSTALL_DIR/autoupdate/registered"
+    if [ -f "$LIST" ]; then
+
+        # Filter list according to
+        # - non-existing repos
+        # - already installed
+        # - if marked as single install.
+        TEMP_LIST=$(mktemp) # temporary list for user
+        NEW_LIST=$(mktemp)  # the new list
+        IFS="$IFS_NEWLINE"
+        while read -r INSTALLED_REPO; do
+            unset IFS
+
+            if [ ! -d "$INSTALLED_REPO" ] ||
+                echo "$EXISTING_REPOSITORY_LIST" | grep -q "$INSTALLED_REPO"; then
+                # Add to new list
+                echo "$INSTALLED_REPO" >>"$NEW_LIST"
+            elif (cd "$INSTALLED_REPO" && "$(git config --local githooks.single.install)" = "yes"); then
+                # Remove autoupdate registered flag from single installs
+                (cd "$INSTALLED_REPO" && git config --local --unset githooks.autoupdate.registered >/dev/null 2>&1)
+            else
+                # Add to list for user
+                echo "$INSTALLED_REPO" >>"$TEMP_LIST"
+            fi
+
+            IFS="$IFS_NEWLINE"
+        done <"$LIST"
+
+        if [ -s "$TEMP_LIST" ]; then
+
+            echo "The following registered repositories in \`$LIST\`"
+            echo "contain also a Githooks installation and have not been updated:"
+            sed -e "s/^/ - /" <"$TEMP_LIST"
```

Comment:
I'm not sure if `sed -e` is defined on OS X (`sed -E` is I believe).
On that note though, you could avoid running `sed` here if you'd do this a few lines up:
```
                # Add to list for user
                echo " - $INSTALLED_REPO" >>"$TEMP_LIST"
```